src/app/loadout/loadout-popup.scss

```diff
@@ -58,10 +67,18 @@
       color: #222;
 
       &.delete {
-        background-color: #ff9294;
+        background-color: $red;
+        color: white;
       }
     }
   }
+
+  > span:last-child {
```

Comment:
what is this meant to apply to?

the only thing it seems to end up applying to on my screen, is
![image](https://user-images.githubusercontent.com/68782081/95003944-ce0c9b80-0599-11eb-8518-848bf95d9c2b.png)
which gives it lopsidedly too much right margin

Reply:
it's not moving the space, weapons, or pencil buttons if that's what it targets

Reply:
It applies to multiple buttons in a row (which we don't do anymore). But if we did do them again, they'd show up as a sort of segmented button instead of separated ones.

Reply:
ah okie doke. an exception for light level would be good, this PR let me know we are cutting it a little too close there. apparently our lowest desktop media increment makes it ellipsisize

Reply:
OK I ripped those back out as they weren't really being used. For the light level we might want to just go back to a single number... or maybe just get rid of the number at all and have people trust us that we can really get to max power?